contracts/InterchainTokenFactory.sol

```diff
@@ -216,8 +207,10 @@ contract InterchainTokenFactory is IInterchainTokenFactory, ITokenManagerType, M
     }
 
     /**
-     * @notice The deployRemoteInterchainToken with originalChainName allows deploying a remote interchain token back to the same chain, which could cause issues with ITS Hub balance tracking.
-     * To fix this, the deployRemoteInterchainToken function was overloaded to prevent self-deployemnt while still allowing self-transfers.
+     * @notice Deploys a remote interchain token on a specified destination chain.
```

Comment:
Is there a `@deprecated` notion in natspec?
```suggestion
     * @notice Deploys a remote interchain token on a specified destination chain. This method is deprecated and will be removed in the future. Please use the above method instead.
```

Reply:
There's no `@deprecated` notion in Natspec: https://docs.soliditylang.org/en/latest/natspec-format.html#tags
Updated it as the suggestion.